release-repo-scripts/merge_forward_pull_request.bash

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+# Copyright (C) 2022 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# The script will merge forward and open a pull request.
```

Comment:
When I first read this I understood that the `git` calls would be handled by the script

```suggestion
# The script will open a pull request for a forward port.
```

It would be nice if these lines were handled by the script


```
git checkout main
git pull
git checkout -b merge67
git merge ign-math6
# manually resolve conflicts
git merge --continue
```

Reply:
I applied your suggestion in https://github.com/gazebo-tooling/release-tools/pull/720/commits/d66e26c205908226e5361a3995cf6ea4595345ca

> It would be nice if these lines were handled by the script

I hear you but I'm not sure how to handle manually resolving conflicts with a script like this. I think it would be easier to have a separate script for the first few lines and then invoke this script afterwards once the conflicts are resolved. At any rate, I think that would be easier to handle in a follow-up pull request

Reply:
A follow-up PR sounds good. 

One idea would be to stop and prompt the user to manually inspect the files and conflicts, similar to

https://github.com/gazebo-tooling/release-tools/blob/b213ef89f81bd4320e13c4ec06342613a28a26a4/release-repo-scripts/bump_dependency.bash#L194-L200

And when the user does `y`, then proceed with the `git merge --continue`.